cpu/cc2538/radio/cc2538_rf_radio_ops.c

```diff
@@ -0,0 +1,471 @@
+#include <errno.h>
+#include <stdio.h>
+
+#include "net/gnrc.h"
+
+#include "cc2538_rf.h"
+#include "cc2538_rf_internal.h"
+
+#include "net/ieee802154/radio.h"
+
+#define CC2538_ACCEPT_FT_2_ACK     (1 << 5)
+
+#if IS_USED(MODULE_IEEE802154_RADIO_HAL)
+static const ieee802154_radio_ops_t cc2538_rf_ops;
+
+ieee802154_dev_t cc2538_rf_dev = {
+    .driver = &cc2538_rf_ops,
+};
+
+static uint8_t cc2538_min_be = 3;
+static uint8_t cc2538_max_be = 5;
+static int cc2538_retries = 4;
```

Comment:
That's retransmissions, right? 
And wasn't the default 3? (Do we have a define for that?)

Reply:
nope, this is the number of CSMA-CA retries (not the number of frame retransmissions). The standard defines Min_BE=3, Retries=4, Max_BE=5 for CSMA-CA params. Maybe this should be included in `net/ieee802154.h`

Reply:
And better add a `cca` somewhere in the name to avoid that confusion.

Reply:
done. I also added the parameters to `net/ieee802154.h`